examples/request/dcql_ac_vc_sd.json

```
@@ -0,0 +1,20 @@
+{
+  "credentials": [
+    {
+      "id": "example_vc_ac_sd",
+      "format": "ac_vp",
+      "meta": {
+        "proof_type_values": ["CLSignature2019"],
+        "schema_id_values": ["did:indy:idu:test:3QowxFtwciWceMFr7WbwnM:2:BasicScheme:0.1"]
```

The structure is not valid for `did:indy`

```suggestion
        "schema_id_values": ["did:indy:idu:test:3QowxFtwciWceMFr7WbwnM/anoncreds/v0/SCHEMA/BasicScheme/0.1"]
```